cmd/calendar.go

```diff
@@ -0,0 +1,13 @@
+package cmd
```

Comment:
Was briefly reading about Cobra before reviewing this PR and was just curious if you used the [Cobra Generator](https://github.com/spf13/cobra#using-the-cobra-generator) to auto-generate these files or if you created the recommended directory structure by hand?

Reply:
I didn't - this was just done by hand.